plugins/nomad/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@backstage/plugin-nomad",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "workspace:^",
+    "@backstage/config": "workspace:^",
+    "@backstage/core-components": "workspace:^",
+    "@backstage/core-plugin-api": "workspace:^",
+    "@backstage/plugin-catalog-react": "workspace:^",
+    "@backstage/theme": "workspace:^",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.60",
+    "react-use": "^17.2.4"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0",
+    "react-router-dom": "*"
+  },
+  "devDependencies": {
+    "@backstage/cli": "workspace:^",
+    "@backstage/core-app-api": "workspace:^",
+    "@backstage/dev-utils": "workspace:^",
+    "@backstage/test-utils": "workspace:^",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^14.0.0",
+    "@types/node": "*",
+    "cross-fetch": "^3.1.5",
+    "msw": "^1.0.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "configSchema": {
+    "$schema": "https://backstage.io/schema/config-v1",
+    "title": "@backstage/nomad",
+    "type": "object",
+    "properties": {
+      "nomad": {
+        "type": "object",
+        "properties": {
+          "addr": {
+            "type": "string",
+            "description": "The address of the Nomad API. See: https://developer.hashicorp.com/nomad/api-docs#addressing-and-ports",
+            "visibility": "frontend"
```

Comment:
based on your comment in the backend README, does the frontend or the backend use this config setting? it almost seems to me like this should not have a visibility frontend, if all frontend does is call the backend, and then it's the backend which hits the Nomad API?  unless i'm misunderstanding the flow!

Reply:
Agree that this is a bit wonky. The reason it's frontend, despite being used (also) by the backend to issue the API calls, is because the `nomad.addr` is used to route to the Nomad `ui` by appending `/ui` to its path in links in the components: https://github.com/backstage/backstage/blob/16c48006ff162274eede499718900aac75827929/plugins/nomad/src/components/EntityNomadAllocationListTable/EntityNomadAllocationListTable.tsx#L47

When followed, it'll open the corresponding page in the [Nomad UI](https://developer.hashicorp.com/nomad/api-docs/ui)